src/mono/wasm/build/WasmApp.targets

```diff
@@ -39,13 +39,19 @@
 
       <_WasmAssemblies Remove="@(_WasmAssemblies)" />
     </ItemGroup>
+    <PropertyGroup>
+      <AOTMode>LLVMOnly</AOTMode>
+      <AOTMode Condition="'$(BuildAOTProfiled)' == 'true'">AotInterp</AOTMode>
+    </PropertyGroup>
 
     <MonoAOTCompiler
       CompilerBinaryPath="$(MicrosoftNetCoreAppRuntimePackRidDir)native\cross\mono-aot-cross"
-      Mode="LLVMOnly"
+      Mode="$(AOTMode)"
       OutputType="AsmOnly"
       Assemblies="@(AotInputAssemblies)"
       UseAotDataFile="false"
+      AotProfilePath="$(AOTProfilePath)"
```

Comment:
Is it valid to have `AOTMode==AotInterp`(due to `BuildAOTProfiled==true`), and empty `AOTProfilePath`? If not, then we should check for that in the task.

Reply:
Is it not possible to build in mixed mode without  a profile?

Reply:
@vargaz can answer that question